pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -25,14 +25,14 @@ in {
   www = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
     pname = "buildbot_www";
-    version = "0.9.7";
+    version = "0.9.9.post1";
 
     # NOTE: wheel is used due to buildbot circular dependency
```

Comment:
Please change your tone in pull request reviews. This is not the linux ml. The semantics of `format` is documented in `buildPythonPackage`.